README.md

```diff
@@ -25,8 +25,14 @@ relevant information. For example, FLFM is ideal for capturing the transient act
 pip install git+https://github.com/ssec-jhu/flfm.git
 ```
 
-See [Usage](#usage) for quick and easy usage instructions.
+See [Usage](#usage) for quick and easy usage instructions for this Python package.
 
+To start using the local GUI application run:
+```term
+pip install -r requirements/app.txt
+python flfm/cli.py app --host=127.0.0.1 --port=8080
```

Comment:
I think for the quick start you may want to drop the extra arguments. Before seeing the usage section, a reader may think they are required.